### PR TITLE
ACD relative_contact_rate

### PR DESCRIPTION
**Pull request by cjbayesian**

### CODE REVIEW

**Comment by cranedroesch** [comment]
Text is up, including stuff in the model info section and stuff in parameter guidance.

File: app.py [hunk 1]
```diff
@@ -86,7 +92,9 @@
 Hospitalizations (**{current_hosp}**), Hospitalization rate (**{hosp_rate:.0%}**), Region size (**{S}**), 
 and Hospital market share (**{Penn_market_share:.0%}**).
 
-A doubling time of {doubling_time} days and a recovery time of {recovery_days} days imply an $R_0$ of {r_naught:.2f}.
+An initial doubling time of {doubling_time} days and a recovery time of {recovery_days} days imply an $R_0$ of
```
Comment (cjbayesian) [inline app.py]: Can we make the dynamic values **bold** to be consistent with above?

File: app.py [hunk 2]
```diff
@@ -370,6 +394,7 @@ def additional_projections_chart(i: np.ndarray, r: np.ndarray) -> alt.Chart:
     """* **Hospitalized COVID-19 Patients:** The number of patients currently hospitalized with COVID-19. This number is used in conjunction with Hospital Market Share and Hospitalization % to estimate the total number of infected individuals in your region.
 * **Currently Known Regional Infections**: The number of infections reported in your hospital's catchment region. This input is used to estimate the detection rate of infected individuals. 
 * **Doubling Time (days):** This parameter drives the rate of new cases during the early phases of the outbreak. The American Hospital Association currently projects doubling rates between 7 and 10 days. The doubling time will be higher (slower spread) or lower (faster spread) depending on the amount of social distancing in your region.
```
Comment (cjbayesian) [inline app.py]: Not sure how to re-write this, but we should change it to make it clear that this should be the estimated doubling time **in the absence** of any social distancing measures.

File: app.py [hunk 3]
```diff
@@ -370,6 +394,7 @@ def additional_projections_chart(i: np.ndarray, r: np.ndarray) -> alt.Chart:
     """* **Hospitalized COVID-19 Patients:** The number of patients currently hospitalized with COVID-19. This number is used in conjunction with Hospital Market Share and Hospitalization % to estimate the total number of infected individuals in your region.
 * **Currently Known Regional Infections**: The number of infections reported in your hospital's catchment region. This input is used to estimate the detection rate of infected individuals. 
 * **Doubling Time (days):** This parameter drives the rate of new cases during the early phases of the outbreak. The American Hospital Association currently projects doubling rates between 7 and 10 days. The doubling time will be higher (slower spread) or lower (faster spread) depending on the amount of social distancing in your region. 
+* **Increase in social distancing:** This parameter allows users to explore how reduction in interpersonal contact might slow the doubling time during the early stages of the outbreak.  Note that we can't say how much any given policy might affect social distance, such as school closures or remote work.
```
Comment (cjbayesian) [inline app.py]: How about "[...] might slow the rate of new infections.  It is your estimate of how much social contact reduction is being achieved in your region relative to the status quo. While it is unclear how much any given policy might affect social contact (eg. school closures or remote work), this parameter lets you see how projections change relative to percentage reductions.

**Comment by cjbayesian** [comment]
I ended up making the changes in my review comments.

**Pull request merged**